pyro/distributions/torch/dirichlet.py

```diff
@@ -52,7 +50,4 @@ def batch_log_pdf(self, x):
         :return: log probability densities of each element in the batch.
         :rtype: torch.autograd.Variable of torch.Tensor of dimension 1.
         """
-        batch_log_pdf = self.torch_dist.log_prob(x).view(self.batch_shape(x) + (1,))
-        if self.log_pdf_mask is not None:
-            batch_log_pdf = batch_log_pdf * self.log_pdf_mask
-        return batch_log_pdf
+        return super(Dirichlet, self).batch_log_pdf(x)
```

Comment:
Do you think it's worth simply removing these docs-only methods, now that we have `copy_docs_from`? It would certainly reduce our codebase.

Reply:
Yes, that's the plan finally, i.e. only class level documentation if needed. The remaining should just be replicated from Distribution or TorchDistribution.

Reply:
Great. I'm also trying to streamline our docs in #708 .